### ISSUE TITLE
Client isn't passed to associations correctly

### ISSUE THREAD

**Issue by cantino** [mavenlink/mavenlink_gem#17]
See http://stackoverflow.com/questions/29288253/ruby-instance-variable-across-classes-or-a-better-way-to-share-an-object-among-c

**Comment by cantino** [comment]
@adambedford I assume it was you posting on Stack Overflow?

**Comment by adambedford** [comment]
@cantino Yes it was. I still haven't figured out a good solution for this

**Comment by cantino** [comment]
We should be able to pass the client threw when we're building associations.  I'd definitely call this a bug.

**Comment by cantino** [comment]
You identified the underlying issue correctly: you only get to assign one token to the `Mavenlink` client.  We will look at fixing that.

**Comment by einzige** [comment]
this is easy thing to do :)

**Comment by cantino** [comment]
Yea, we can fix it.

**Comment by cantino** [comment]
@adambedford Is this fixed for you now?
